www/js/diary/infinite_scroll_trip_item.js

```diff
@@ -0,0 +1,159 @@
+    // };
+
+    // Converting trip to tripgj
+    Timeline.confirmedTrip2Geojson($scope.trip).then((tripgj) => {
+      $scope.$apply(() => {
+          $scope.tripgj = $scope.trip;
+          $scope.tripgj.data = tripgj;
+          $scope.tripgj.common = {};
+          $scope.tripgj.common.earlierOrLater = '';
+          $scope.tripgj.pointToLayer = DiaryHelper.pointFormat;
+
+          console.log("Is our trip a draft? ", DiaryHelper.isDraft($scope.tripgj));
+          $scope.tripgj.isDraft = DiaryHelper.isDraft($scope.tripgj);
+          console.log("Tripgj == Draft: ", $scope.tripgj.isDraft);
+
+          console.log("Tripgj in Trip Item Ctrl is ", $scope.tripgj);
+
+          // var tc = getTripComponents($scope.tripgj);
+          // $scope.tripgj.sections = tc[3];
+          // $scope.tripgj.percentages = DiaryHelper.getPercentages($scope.trip);
+          // console.log("Section Percentages are ", $scope.tripgj.percentages);
+      });
+    });
+    console.log("Trip's Date is ", $scope.trip.display_date);
+    console.log("Trip in Trip Item Ctrl is ", $scope.trip);
+
+    // Explain Draft Function for the button to explain what a draft is
+    $scope.explainDraft = function($event) {
+      $event.stopPropagation();
+      $ionicPopup.alert({
+        template: $translate.instant('list-explainDraft-alert')
+      });
+      // don't want to go to the detail screen
+    }
+
+    // In-Line Map, functionality pulled from Infinite Scroll Detail
+    $scope.mapCtrl = {};
+    angular.extend($scope.mapCtrl, {
+      defaults : {
+        zoomControl: false
+      }
+    });
+
+    angular.extend($scope.mapCtrl.defaults, Config.getMapTiles())
+
+    var mapEvents = leafletMapEvents.getAvailableMapEvents();
+    for (var k in mapEvents) {
+      var eventName = 'leafletDirectiveMap.infscroll-tripitem.' + mapEvents[k];
+      $scope.$on(eventName, function(event, data){
+          try {
+              console.log("in mapEvents, event = "+JSON.stringify(event.name)+
+                    " leafletEvent = "+JSON.stringify(data.leafletEvent.type)+
+                    " leafletObject = "+JSON.stringify(data.leafletObject.getBounds()));
+          } catch (e) {
+              if (e instanceof TypeError) {
+                  console.log("in mapEvents, event = "+JSON.stringify(event.name)+
+                        " leafletEvent = "+JSON.stringify(data.leafletEvent.type)+
+                        " leafletObject is undefined");
+              } else {
+                  console.log(e);
+              }
+          }
+          $scope.eventDetected = event.name;
+      });
+    }
+
+    $scope.refreshTiles = function() {
+      $scope.$broadcast('invalidateSize');
+    }
+
+    $scope.$on('leafletDirectiveMap.infscroll-tripitem.resize', function(event, data) {
+      console.log("detail received resize event, invalidating map size");
+      data.leafletObject.invalidateSize();
+    });
+});
```

Comment:
I don't see where `leafletDirectiveMap.infscroll-tripitem.' + mapEvents[k]` is _emitted_, only where we listen for it with `$event.$on`

Reply:
part of the reason that I am concerned about this is because `invalidateSize` on refresh can actually sometimes break the map display. We've tried to fix this issue for years, and I finally spent a week or so digging into it and found an issue with leaflet in which calling `invalidateSize` when the map is hidden can cause the grayed out/missing tiles.

https://github.com/e-mission/e-mission-phone/commit/2b724685b9de4b98d139eefa85c0fcc1bc3bf4b2

So I definitely don't want to add more `invalidateSize` calls that I have to.

Reply:
This code was added into the new file [`www/js/diary/infinite_scroll_trip_item.js` in Tyler's Unified diary screen PR](https://github.com/tyleryandt18/e-mission-phone/blame/0925801d6081e2675ba5b3071afcab50c4af514e/www/js/diary/infinite_scroll_trip_item.js). There is no comment that explicitly states the purpose of this code snippet.

Because of this, I investigated this by searching for the line of code "`data.leafletObject.invalidateSize();`" and I found that there is another file with almost identical code - 

**Here is our file: `www/js/diary/infinite_scroll_trip_item.js`**
<img width="1346" alt="Screen Shot 2022-09-26 at 9 59 35 AM" src="https://user-images.githubusercontent.com/61334340/192326093-04af9433-a2f1-4fac-8eda-368bd2baea81.png">


**Here is the original file: `www/js/diary/detail.js`** 
<img width="1344" alt="Screen Shot 2022-09-26 at 10 13 23 AM" src="https://user-images.githubusercontent.com/61334340/192327590-6bd9bbfa-cf44-4df7-81fa-e82da46c2e0d.png">


The code in the original file appears to be added in completely new in [Shankari's Commit here "File the loading on Android"](https://github.com/tyleryandt18/e-mission-phone/commit/19282edf171c7e3dd7db9dcf987ea997c8f35dc0) as _"In this change, I apply the workaround (for the issue of maps on android frequently ending up with tiles only partially loaded) automatically and for all maps"_. 

**Since this came from code that is relatively unrelated to the android map tiles not loading issue, I believe we can remove this code safely without it affecting the functionality of the infinite scroll. _I tested this out on Android Pixel 6 Emulator and iPhone 14 Pro Max emulator and both appeared to display just fine without the code:_**
<img width="897" alt="Screen Shot 2022-09-26 at 10 25 09 AM" src="https://user-images.githubusercontent.com/61334340/192330531-6b072f63-4d98-4f17-a154-483715603ebf.png">

Reply:
I have removed this code in my commit - Removing unnecessary leaflet Map invalidateSize() function

Reply:
This is fine.
For the record, the "android map tiles not showing" issue does not reproduce easily in the emulator. It would be so much easier to debug in that case 😄 
But given that this does not appear to be added intentionally, it is best to remove it.